app/localization.js

```diff
@@ -10,36 +10,34 @@
  * Localization Utilities
  */
 
-export function Localizer() {
-    // Currently configured language
-    this.language = 'en';
+export class Localizer{
+    constructor() {
+        // Currently configured language
+        this.language = 'en';
 
-    // Current dictionary of translations
-    this.dictionary = undefined;
-}
+        // Current dictionary of translations
+        this.dictionary = undefined;
+    }
 
-Localizer.prototype = {
     // Configure suitable language based on user preferences
-    setup: function (supportedLanguages) {
-        var userLanguages;
+    setup(supportedLanguages) {
 
         this.language = 'en'; // Default: US English
 
         /*
          * Navigator.languages only available in Chrome (32+) and FireFox (32+)
          * Fall back to navigator.language for other browsers
          */
-        if (typeof window.navigator.languages == 'object') {
-            userLanguages = window.navigator.languages;
-        } else {
-            userLanguages = [navigator.language || navigator.userLanguage];
-        }
 
-        for (var i = 0;i < userLanguages.length;i++) {
-            var userLang = userLanguages[i];
-            userLang = userLang.toLowerCase();
-            userLang = userLang.replace("_", "-");
-            userLang = userLang.split("-");
+        const userLanguages = (typeof window.navigator.languages == 'object')
+            ? window.navigator.languages
+            : [navigator.language || navigator.userLanguage];
```

Comment:
This could be a bit excessive style enforcement. Can we turn such things off?

Reply:
This is not really enforced. I change it so user lang is a const.

As I said below, most ES6 good practices guidelines recommend using const whenever possible.

Reply:
Ah, I missed the const thing. That's fine in that case. But for clarity it would be good to separate the things you manually restructured to get things to be `const` and the things `eslint` automatically converted.